### ISSUE TITLE
Does not work on test directory in mix project

### ISSUE THREAD

**Issue by dylan-chong** [rrrene/credo#487]
### Precheck

* For proposing a new feature, be sure to name your issue "Proposal: ..."
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

```
> mix credo -v
0.8.8

> elixir -v
Erlang/OTP 20 [erts-9.2] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Elixir 1.5.3
```

* Operating system: mac 10.13.2

### What were you trying to do?

Use `mix credo` to lint my test files in `test/`.

### Expected outcome

 `mix credo` should show warnings that `mix credo --read-from-stdin < $testfile` does.

### Actual outcome

 `mix credo` does not show warnings that `mix credo --read-from-stdin < $testfile` does.

**Comment by NobbZ** [comment]
You need to add your test directory under the `configs.files.included` key in your `.credo.exs`, eg:

```exs
# config/.credo.exs
%{
  configs: [
    %{
      name: "default",
      files: %{
        included: ["lib/", "src/", "web/", "apps/", "test/"],
        excluded: []
      },
      checks: [
        # ... several checks omitted for readability ...
      ]
    }
  ]
}
```

More info about that file and how generate one with default values cann be read at https://github.com/rrrene/credo#configuration-via-credoexs

**Comment by dylan-chong** [comment]
Looks like the test directory isn't included in the generated file by default. it seems a bit counterintuitive like that. Could i request a change?

**Comment by rrrene** [comment]
@dylan-chong I would accept a PR for that :+1:
